mu/interface.py

```diff
@@ -755,6 +760,30 @@ def __init__(self, port, theme='day', parent=None):
             raise IOError("Cannot connect to device on port {}".format(port))
         self.set_theme(theme)
 
+    def repl_paste(self):
+        """
+        Grabs clipboard contents then sends down the serial port
+        """
+        clipboard = QApplication.clipboard()
+        self.serial.write(bytes(clipboard.text(), 'utf8'))
```

Comment:
Will this throw an exception if the clipboard contains something other than text? If it returns something like `None`, or an empty string, does how does the micropython process that?